File: rad_models/radpackages.js
/* jshint indent: 1 */

export default function (sequelize, _DataTypes) {
  return sequelize.define(
    "radpackages",
    {
      id: {
        type: "INT(11)",
        allowNull: false,
        defaultValue: null,
        primaryKey: true,
        autoIncrement: true,
        comment: null,
        foreignKey: [Object],
      },
      groupname: {
        type: "VARCHAR(64)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
        foreignKey: [Object],
      },
      daily_volume: {
        type: "INT(12)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      weekly_volume: {
        type: "INT(12)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      monthly_volume: {
        type: "INT(12)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      priority: {
        type: "INT(12)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      session: {
        type: "INT(11)",
        allowNull: false,
        defaultValue: "3",
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
    },
    {
      tableName: "radpackages",
    }
  );
}

File: internet/model/radpackages.py
from sqlalchemy.orm import Mapped, mapped_column

from . import Base


# pyre-ignore[11]
class RadiusPackages(Base):
    __tablename__ = "radpackages"

    id: Mapped[int] = mapped_column(primary_key=True)
    group_name: Mapped[str] = mapped_column("groupname")
    daily_volume: Mapped[int] = mapped_column("daily_volume")
    weekly_volume: Mapped[int] = mapped_column("weekly_volume")
    monthly_volume: Mapped[int] = mapped_column("monthly_volume")
    priority: Mapped[int] = mapped_column("priority")
    session: Mapped[int] = mapped_column("session")
